src/components/Navbar.js

```diff
@@ -1,4 +1,3 @@
-import React from 'react';
 import { Link } from 'react-router-dom';
 import './Navbar.css';
```

Comment:
Hi @FrederickMih,

Thanks for the feedback. I have one issue with making all the padding and margins EXACTLY the same as the original design because we can no longer access the website. If you try to log in with the provided credentials, it is no longer working now. So, how can I get the EXACT measurements now?

Reply:
The same goes with the Category section. I am not able to see the reference

Reply:
I kept trying and trying and it looks like the credentials are working again now after a couple of hours not working. Alright, I'll see what I can do. But if I may, where can I see the Categories page? There's only 1 design and it's the books page.

Reply:
Well Resolved @JauJau088 👍🏽

Reply:
Thank you @Whoistolu & @FrederickMih